src/main/resources/application.yml

```diff
@@ -22,6 +22,9 @@ client:
   vsac_base_url: https://vsac.nlm.nih.gov/vsac
   valueset_endpoint: /svs/RetrieveMultipleValueSets?id={oid}&profile={profile}&includeDraft={includeDraft}
   default_profile: ${DEFAULT_VSAC_PROFILE:Most Recent Code System Versions in VSAC}
+  fhir-terminology-service:
+    base-url: https://uat-cts.nlm.nih.gov/fhir
```

Comment:
this url seems temporary. should we introduce env var for this?

Reply:
Yes, good catch, I added a new variable. Will keep Brandon updated